### ISSUE TITLE
Separate Minishift and Minikube instructions in two documents

### ISSUE THREAD

**Issue by rhuss** [redhat-developer-demos/knative-tutorial#23]
Currently having both, minikube and minishift in the instructions is confusing for the reader:

* E.g. in Setup, both "minikube" and "minishift" are added to the list of prerequisite tools, suggesting that *both* are needed at the same time. But that's not true, you need either-or.
* In the instructions having always oc and kubectl pairwise is super distracting, even when they are separated with an "(OR)" instruction. 

I'm very sure that a reader from the very beginning decides, whether she wants to follow the tutorial with minikube **or** minishift, so this decision happens very early. Having then all the commands for the _other_ system to read is disturbing.

### Suggestions:

* Either create two sets of instructions. This can be easily done with asciidoc and parameters and two runs to create two different sets of instructions. One for minikube, one for minishift. That way you can keep one source but two outputs. 
* In the sample code for kubectl/oc, use two buttons top right in the code box to switch between the commands. This is not that nice than having two different documentation but could work, too.

tldr; I highly recommend to not intermix Kubernetes and OpenShift commands/prerequisites as the user will decide from the very beginning what platform to use.

**Comment by kameshsampath** [comment]
@rhuss  thanks for the review and feedback. 

> E.g. in Setup, both "minikube" and "minishift" are added to the list of prerequisite tools, suggesting that both are needed at the same time. But that's not true, you need either-or.

This depends on the comfort of the developer, actually am thinking to just have minikube only for upstream as we will not have real OpenShift based product until OCP 4, where minishift is under question

> In the instructions having always oc and kubectl pairwise is super distracting, even when they are separated with an "(OR)" instruction. 

Yeah I like the idea of codebox than two different instructions and user can switch. do you have an example that I can see how to do it ?

**Comment by kameshsampath** [comment]
introduced tabs that differentiates the instructions
